specificatie/persoon.yaml

```diff
@@ -105,6 +111,16 @@ components:
           $ref: 'overlijden.yaml#/components/schemas/GbaOverlijden'
         verblijfplaats:
           $ref: 'verblijfplaats.yaml#/components/schemas/GbaVerblijfplaats'
+        immigratie:
+          $ref: 'immigratie.yaml#/components/schemas/GbaImmigratie'
+        gemeenteVanInschrijving:
+          $ref: 'common.yaml#/components/schemas/Waardetabel'
+        datumAanvangAdreshouding:
```

Comment:
waarom zit deze niet in verblijfplaats?

Reply:
Verplaatst naar verblijplaats.yaml (gbaverblijfplaats)